src/System.Management.Automation/engine/ArgumentToVersionTransformationAttribute.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Management.Automation
+{
+    /// <summary>
+    /// To make it easier to specify a version, we add some conversions that wouldn't happen otherwise:
+    ///   * A simple integer, i.e. 2;
+    ///   * A string without a dot, i.e. "2".
+    /// </summary>
+    public class ArgumentToVersionTransformationAttribute : ArgumentTransformationAttribute
```

Comment:
private/internal sealed?

Reply:
make just as internal

`sealed` blocks inheritance and `private` doesn't cover by `InternalsVisibleTo` attribute